src/open_inwoner/openklant/services.py

```diff
@@ -79,3 +84,299 @@ def update_user_from_klant(klant: Klant, user: User):
             f"updated user from klant API with fields: {', '.join(sorted(update_data.keys()))}",
             content_object=user,
         )
+
+
+class OpenKlant2Service:
+
+    client: OpenKlant2Client
+
+    def __init__(self, client: OpenKlant2Client):
+        if not isinstance(client, OpenKlant2Client):
+            raise ValueError(
+                f"`client` must be an instance of {type(OpenKlant2Client)}"
+            )
+        self.client = client
+
+    def find_partij_for_params(self, params: PartijListParams):
+        resp = self.client.partij.list(params=params)
+
+        if (count := resp["count"]) > 0:
+            if count > 1:
+                # TODO: Is this a use-case? The API seems to allow this
```

Comment:
This seems like a valid issue for OK, good that you already raised it: https://github.com/maykinmedia/open-klant/issues/241